providers/provider_default.go

```diff
@@ -116,6 +117,16 @@ func (p *ProviderData) EnrichSession(_ context.Context, _ *sessions.SessionState
 // Authorize performs global authorization on an authenticated session.
 // This is not used for fine-grained per route authorization rules.
 func (p *ProviderData) Authorize(_ context.Context, s *sessions.SessionState) (bool, error) {
+	println("ACR level is", s.Acr)
```

Comment:
Sorry my fault 😅. I develop on Windows and because the OAuth2 Proxy can't compile on it. I always used the Docker image. I removed the debug statements.